domains/client/domain-operator/src/fraud_proof.rs

```diff
@@ -101,6 +102,22 @@ where
         }
     }
 
+    pub fn generate_bad_valid_bundle_proof<PCB>(
```

Comment:
I dont think this function is necessary especilly when its just a initializing the Fraud Proof